opentelekomcloud/resource_opentelekomcloud_rds_instance_v3.go

```diff
@@ -750,14 +750,14 @@ func buildRdsInstanceV3CreateParameters(opts map[string]interface{}, arrayIndex
 		params["security_group_id"] = v
 	}
 
-	v, err = navigateValue(opts, []string{"subnet_id"}, arrayIndex)
+	v, err = navigateValue(opts, []string{"network_id"}, arrayIndex)
 	if err != nil {
 		return nil, err
 	}
 	if e, err := isEmptyValue(reflect.ValueOf(v)); err != nil {
 		return nil, err
 	} else if !e {
-		params["subnet_id"] = v
+		params["network_id"] = v
```

Comment:
I guess here it should stay subnet_id, since the API really expects subnet_id parameter, but with neutron network id content